client/.prettierrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "plugins": ["prettier-plugin-tailwindcss"]
+}
```

Comment:
I noticed that there was only 2 checks ran when looking at the PR. Seems like there was an error running one of the linters coming from this plugin? https://github.com/ubcesports/lounge-hub/actions/runs/11474321655/job/31930023056?pr=42. 

<img width="1037" alt="Screenshot 2024-10-23 at 11 12 02 AM" src="https://github.com/user-attachments/assets/6670330b-7cb0-427a-9178-8e38d030707e">

Reply:
I believe this is because prettier config lives in client and format runs from root, we might need to add "prettier-plugin-tailwindcss" to root `package.json`